src/calculateRentalCost.js

```diff
@@ -28,7 +28,11 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  return days < 3 // if car is rented for less then 3 days
+    ? days * 40 // total cost will be equal to clear cost-per-day times days
+    : days >= 3 && days < 7 // othrwise we'll check if term is between 3 and 6
+      ? (days * 40) - 20 // in this case total cost will be decreased on 20
+      : (days * 40) - 50; // last case - `days` is 7+ - we'll get -$50
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, I don't know what 40, 50, 20, 7 and 3 are

Reply:
you repeat `days * 40` calculation 3 times, create a variable for it

Reply:
fixed all issues